app/models/casa_org.rb

```diff
@@ -28,6 +28,7 @@ class CasaOrg < ApplicationRecord
   has_many :contact_topics
   has_one_attached :logo
   has_one_attached :court_report_template
+  has_many :placement_types, dependent: :destroy
```

Comment:
unless i'm mistaken, there's currently no way for placement types to be created for an org. should i add that work to this PR or should another issue be made?

Reply:
Case placements are part of a larger epic https://github.com/rubyforgood/casa/issues/4695. https://github.com/rubyforgood/casa/issues/5778 that issue deals with the creation of placements at an org level